code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -1,17 +1,17 @@
 /obj/item/projectile/energy/bolt //ebow bolts
 	name = "bolt"
 	icon_state = "cbbolt"
-	damage = 15
-	damage_type = TOX
-	nodamage = FALSE
-	stamina = 60
-	eyeblur = 10
-	knockdown = 10
-	slur = 5
+	irradiate = 200
+	pass_flags = PASSGLASS
+
+/obj/item/projectile/energy/bolt/on_hit(atom/target, blocked = FALSE)
+	..()
+	if(ishuman(target))
```

Comment:
No exciting things on non-humans?

Reply:
ebow is a weapon for fighting humans, yes.